ingestion/src/metadata/utils/connection_clients.py

```diff
@@ -40,3 +40,9 @@ def __init__(self, client) -> None:
 class KafkaClient:
     def __init__(self, client) -> None:
         self.client = client
+
+
+@dataclass
+class RedashClient:
```

Comment:
Please drop a line as docstring in the connection_clients module to explain why we're wrapping all these clients 🙏  We can merge this afterwards. Thanks!